test/conftest.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import annotations
```

Comment:
Only appeared in `3.7.0b1`. This confirms that the Readme should be updated to not support 3.5 and 3.6.